setup.py

```diff
@@ -215,6 +226,25 @@ def agg_extensions():
         '-c++',
     ]
 
+    # Figure out the freetype linkage
+    if sys.platform == 'win32':
+        # XXX: Note that Windows has used FreeType for many years. This would
+        # be a change in behavior.
+        define_macros += [('KIVA_USE_WIN32', None)]
+        agg_include_dirs += [os.path.join(agg_dir, 'font_win32_tt')]
+        kiva_agg_sources += [
+            os.path.join(agg_dir, 'font_win32_tt', 'agg_font_win32_tt.cpp'),
+        ]
```

Comment:
It's important to check that this doesn't change things too much on Windows. If it does, we'll have to figure out how to link against an external FreeType on Windows.

Reply:
Using the Win32 font rendering appears to break text rotation, according to @rahulporuri's testing